sibylapp/view/utils/helpers.py

```diff
@@ -30,14 +30,28 @@ def show_sort_options(options):
     return st.radio("Sort by", options, horizontal=True)
 
 
+def show_text_input_side_by_side(label, default_input=None, numeric=False, **input_params):
+    col1, col2 = st.columns([2, 2])
+    col1.markdown(label)
+
+    if numeric:
+        return col2.number_input(
+            "hidden", value=default_input, label_visibility="collapsed", **input_params
+        )
+    else:
+        return col2.text_input(
```

Comment:
I think this would be much easier to use as a selectbox with all category options from the database - do you think that would be doable with the current setup?

Reply:
I did think about that! Does the database contain information on options for each categorical variable? Otherwise, we need to iterate through the training set and find unique values for each variable.

Reply:
At the moment, you would need to iterate through the training set, which comes at the downside of preventing users from picking something that isn't in the database. I think having the categories (as well as interpretable names for these categories) in the database is something we should add (added issue https://github.com/sibyl-dev/sibyl-api/issues/79).

For now, its up to you if you want to add the iterate-through-dataset implementation. I think whether its worth it depends how much effort that would take.